Code/Tools/TestImpactFramework/Runtime/Python/Code/Source/TestImpactPythonRuntime.cpp

```diff
@@ -146,6 +147,33 @@ namespace TestImpact
 
     PythonRuntime::~PythonRuntime() = default;
 
+    AZStd::pair<AZStd::vector<const PythonTestTarget*>, AZStd::vector<const PythonTestTarget*>> PythonRuntime::SelectCoveringTestTargets(
+        const ChangeList& changeList, Policy::TestPrioritization testPrioritizationPolicy)
+    {
+        AZStd::vector<const TestTarget*> discardedTestTargets;
+
+        // Select and prioritize the test targets pertinent to this change list
+        const auto changeDependencyList = m_dynamicDependencyMap->ApplyAndResoveChangeList(changeList, m_integrationFailurePolicy);
+        const auto selectedTestTargets = m_testSelectorAndPrioritizer->SelectTestTargets(changeDependencyList, testPrioritizationPolicy);
+
+        // Populate a set with the selected test targets so that we can infer the discarded test target not selected for this change list
+        const AZStd::unordered_set<const TestTarget*> selectedTestTargetSet(selectedTestTargets.begin(), selectedTestTargets.end());
+
+        // Update the enumeration caches of mutated targets regardless of the current sharding policy
+        // EnumerateMutatedTestTargets(changeDependencyList);
```

Comment:
Can we remove this commented line? Or can it be uncommented now?

Reply:
Deffo removed. Removed 👍